markdown_xblock/html.py

```diff
@@ -267,30 +279,119 @@ def get_editable_fields(self):
 
         return fields
 
+    @classmethod
+    def load_definition(cls, xml_object, system, location, id_generator):
+        """Load markdown content from file."""
+
+        filename = xml_object.get('filename')
+        pointer_path = "{category}/{url_path}".format(
+            category='markdown',
+            url_path=location.block_id.replace(':', '/')
+        )
+        base = path(pointer_path).dirname()
+        filepath = u"{base}/{name}.md".format(base=base, name=filename)
+
+        try:
+            with system.resources_fs.open(filepath, encoding='utf-8') as infile:
+                markdown = infile.read()
+
+                definition = {'data': markdown}
+
+                return definition
+
+        except ResourceNotFound as err:
+            msg = 'Unable to load file contents at path {0}: {1} '.format(
+                filepath, err)
+            # add more info and re-raise
+            six.reraise(Exception, Exception(msg), sys.exc_info()[2])
```

Comment:
Any particular reason why you can't replace this line with just `raise`? It would save you the `six` import (which we should kind of avoid, as we're only interested in Python 3 anymore).

Reply:
No reason, just a copy-paste-cleanup mishap. Will remove that import.